admin/display/general-metaboxes.php

```diff
@@ -43,7 +43,7 @@ static function display_extra_metaboxes( $add, $meta ) {
 							<strong>
 								<?php
 								echo aiosp_common::get_upgrade_hyperlink( 'side', __( 'Pro Version', 'all-in-one-seo-pack' ), __( 'CLICK HERE', 'all-in-one-seo-pack' ), '_blank' );
-								echo __( ' to upgrade to Pro Version and get:', 'all-in-one-seo-pack' );
+								printf( ' %1$s', __( 'to upgrade to Pro Version and get:', 'all-in-one-seo-pack' ) );
```

Comment:
When there's only one placeholder, there's no need to use %1$s, just %s is fine.